src/rgw/rgw_rest_s3.cc

```diff
@@ -463,6 +463,15 @@ int RGWGetObj_ObjStore_S3::send_response_data(bufferlist& bl, off_t bl_ofs,
       }
     } catch (const buffer::error&) {} // omit x-rgw-replicated-from headers
   }
+  if (auto i = attrs.find(RGW_ATTR_OBJ_REPLICATION_TIMESTAMP);
+      i != attrs.end()) {
+    try {
+      ceph::real_time replicated_time;
+      decode(replicated_time, i->second);
+      dump_time(s, "x-rgw-replicated-at", replicated_time);
```

Comment:
@smanjara i think we want to use `dump_time_header()` here instead? `dump_time()` is calling `s->formatter->dump_string()` to format this as json/xml

i'm tracking a regression here in https://tracker.ceph.com/issues/65373